configure.ac

```diff
@@ -405,6 +405,18 @@ PKG_CHECK_MODULES([JSON],[json-c],AC_CHECK_LIB(
 
 OPTIONAL_JSONC_LIBS="${JSONC_LIBS}"
 
+# -----------------------------------------------------------------------------
+# libbpf
+
```

Comment:
You added the `libbpf` here and I think we also need to remove the reference to `libdl` that was inserted inside this file.